Encounters.md

```diff
@@ -0,0 +1,65 @@
+An `encounter` in ID3C is "[An interaction with an individual to collect point-in-time information or samples](https://github.com/seattleflu/id3c/blob/c5ee5b8d9dbd87a89213f5044a1632cecefd4e7f/schema/deploy/warehouse/encounter.sql#L25)."
+An associated [Individual] and Site are required to create an Encounter (see [warehouse schema diagram]).
+
+## REDCap projects
+In addition to requiring sufficient information to create an associated [Individual] and Site (i.e. physical place of encounter), we wait to create Encounters in ID3C until the following conditions are met for each project:
+
+### [SCAN]
+- Mail
+  - `consent_form` is complete and verified
+  - `enrollment_questionnaire` is complete and verified
+  - `illness_q_date` is not blank or `back_end_mail_scans` is complete and verified
+  - `post_collection_data_entry_qc` is complete and verified and we can create an associated [Sample], or `post_collection_data_entry_qc` is not complete and verified
```

Comment:
Not required for an encounter to be created. Only required to create a sample.

Reply:
If `specimen_received` is complete but `specimen_entry` cannot be made (i.e. we can't create an associated Sample), then we "[Skip] enrollment with insufficent [sic] information to construct a specimen".

Reply:
But we also ingest the encounter data without sample if `post_collection_data_entry_qc` is not complete.